Tests/UnitTests/Core/Seeding/HoughTransformTest.cpp

```diff
@@ -0,0 +1,135 @@
+
+BOOST_AUTO_TEST_CASE(hough_transform_seeder) {
+  // we are using the slope on yz plane with the y coordinate (hardcoded from
+  // the csv MuonSimHit data)
+  std::vector<std::pair<double, double>> simHits = {
+      {-0.0401472 / 0.994974, -422.612}};
+  // The drift circles info (hardcoded from the csv file)
+  std::vector<std::vector<float>> tubePos{{-4.54747e-13, -427.981, -225.541},
+                                          {0, -412.964, -199.53},
+                                          {1.36424e-12, -427.981, -173.519},
+                                          {-9.09495e-13, -427.981, 173.519},
+                                          {0, -442.999, 199.53},
+                                          {-4.54747e-13, -427.981, 225.541}};
+  std::vector<float> driftRadius{14.5202, 1.66237, 12.3176,
+                                 1.5412,  12.3937, 3.77967};
+  std::vector<std::vector<std::int8_t>> stationInfo{
+      {5, -1, 4, 1, 1, 39}, {5, -1, 4, 1, 2, 38}, {5, -1, 4, 1, 3, 39},
+      {5, -1, 4, 2, 1, 39}, {5, -1, 4, 2, 2, 39}, {5, -1, 4, 2, 3, 39}};
+
+  BOOST_CHECK_EQUAL(tubePos.size(), driftRadius.size());
+  BOOST_CHECK_EQUAL(stationInfo.size(), tubePos.size());
+  // Define the drift Circles
+  ActsExamples::DriftCircleContainer driftCircles;
+  for (std::size_t i = 0; i < tubePos.size(); i++) {
+    ActsFatras::Hit::Vector3 tube_pos{tubePos[i][0] * Acts::UnitConstants::mm,
+                                      tubePos[i][1] * Acts::UnitConstants::mm,
+                                      tubePos[i][2] * Acts::UnitConstants::mm};
+    driftCircles.push_back(ActsExamples::DriftCircle(
+        std::move(tube_pos), driftRadius[i], 0.0f, stationInfo[i][0],
+        stationInfo[i][1], stationInfo[i][2], stationInfo[i][3],
+        stationInfo[i][4], stationInfo[i][5]));
+  }
+
+  // configure the binning of the hough plane
+  Acts::HoughTransformUtils::HoughPlaneConfig planeCfg;
+  planeCfg.nBinsX = 1000;
+  planeCfg.nBinsY = 1000;
+
+  // instantiate the peak finder
+  Acts::HoughTransformUtils::PeakFinders::IslandsAroundMaxConfig peakFinderCfg;
+  peakFinderCfg.fractionCutoff = 0.7;
+  peakFinderCfg.threshold = 3.;
+  peakFinderCfg.minSpacingBetweenPeaks = {0., 30.};
+
+  // and map the hough plane to parameter ranges.
+  // The first coordinate is tan(theta), the second is z0 in mm
+  Acts::HoughTransformUtils::HoughAxisRanges axisRanges{-3., 3., -2000., 2000.};
+
+  // create the functions parametrising the hough space lines for drift circles.
+  // Note that there are two solutions for each drift circle and angle
+
+  // left solution
+  auto houghParam_fromDC_left = [](double tanTheta,
+                                   const ActsExamples::DriftCircle& DC) {
+    return DC.y() - tanTheta * DC.z() -
+           DC.rDrift() / std::cos(std::atan(tanTheta));
+  };
+  // right solution
+  auto houghParam_fromDC_right = [](double tanTheta,
+                                    const ActsExamples::DriftCircle& DC) {
```

Comment:
here you could use the helper `struct` from my comment above, if you agree to drop the `ActsExamples` dependency